.eslintrc.yml

```diff
@@ -447,19 +529,33 @@ rules:
     - error
   no-var:
     - error
+  object-shorthand:
+    - error
+    - always
   prefer-const:
     - error
   prefer-reflect:
     - error
   prefer-spread:
     - error
+  prefer-numeric-literals:
+    - error
+  prefer-rest-params:
+    - error
+  prefer-template:
+    - error
+  prefer-arrow-callback:
+    - error
+    - allowNamedFunctions: false
```

Comment:
Does this mean `.method( function() {})` or `.on( 'event', function() {})` will be disallowed?